include/auxkernels/Current.h

```diff
@@ -24,6 +24,7 @@ class CurrentTempl : public AuxKernel
   virtual Real computeValue() override;
 
 protected:
+  int _component;
```

Comment:
The _component member variable should be const since it is only set once during construction